ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -132,6 +131,21 @@ tests:
     - ref: stackrox-e2e
     workflow: ipi-aws
   timeout: 5h0m0s
+- always_run: false
+  as: ocp-stable-fips-qa-e2e-tests
```

Comment:
Can you just add a `cluster_claim:` for 4.12 here if you need 4.12?

Reply:
I don't think 4.12 clusters from the pool are FIPS enabled. Though maybe there is a way to enable FIPS post install @BradLugo ?

Reply:
That's correct; it was due to FIPS. My understanding is that the FIPS configuration for OpenShift changes configurations at the OS-level, so I don't think it's possible to do a claim + post. If someone finds documentation or evidence of the contrary, please let me know (it would be rad to use claims rather than the on-demand profiles for this)